e2e/test/scenarios/sharing/public-question.cy.spec.js

```diff
@@ -226,15 +227,31 @@ describe("scenarios > question > public link with extension", () => {
 
   ["csv", "xlsx"].forEach((fileType) =>
     it(`should download a ${fileType} file when a public link with .${fileType} is shared`, () => {
-      downloadPublicFileURL(fileType, (response) => {
-        const { SheetNames, Sheets } = xlsx.read(response.body, {
-          type: "binary",
-        });
+      downloadPublicFileURL(fileType, async (response) => {
+        let rows;
+
```

Comment:
I. need to create a helper to avoid duplicating this. But, do we even need to assert against the file contents? Can we just check that the file exists?